queue/util/createConsumer.js

```diff
@@ -87,8 +65,10 @@ const createConsumer = module.exports = async function (options) {
   })
 
   amqConnection.on('error', _handleConnectionClosed)
-  amqConnection.on('close', _handleConnectionClosed)
 
   await consumer.start()
+
+  logger.info(`Consumer ${consumer.getTag()} is now consuming on queue: ${queueName}`)
```

Comment:
This is super minor, but I've been using quotes around info like this. I think it makes it very clear and kinda removes the need for colons. What do you think?

```js
logger.info(`Consumer "${consumer.getTag()}" is now consuming on queue "${queueName}"`)
```